lib/instances/GuiObject.lua

```diff
@@ -0,0 +1,94 @@
+local BaseInstance = import("./BaseInstance")
+local Color3 = import("../types/Color3")
+local InstanceProperty = import("../InstanceProperty")
+local Signal = import("../Signal")
+local UDim2 = import("../types/UDim2")
+local Vector2 = import("../types/Vector2")
+local GuiObject = BaseInstance:extend("GuiObject")
+
+GuiObject.properties.Active = InstanceProperty.typed("boolean", {
+	getDefault = function()
+		return true
+	end,
+})
+
+
+GuiObject.properties.AnchorPoint = InstanceProperty.typed("Vector2", {
+	getDefault = function()
+		return Vector2.new()
+	end,
+})
+
+GuiObject.properties.AutoButtonColor = InstanceProperty.typed("boolean", {
+	getDefault = function()
+		return false
+	end,
+})
```

Comment:
We should try to implement the properties in a way that lines up with the real inheritance tree. `AutoButtonColor` is only on `TextButton` and `ImageButton`, and defaults to `true` when inserted via `Instance.new`, which has different defaults than when inserted via Studio.

The best reference for that is probably http://wiki.roblox.com/index.php?title=API:Class/GuiObject

Reply:
whoops missed this one. was mocking TOO FAST